packages/extension-browser/package.json

```diff
@@ -7,7 +7,7 @@
     "timeout": "1m"
   },
   "browserslist": "last 2 chrome versions, last 2 firefox versions",
-  "bundleSize": 645000,
+  "bundleSize": 660000,
```

Comment:
This should be able to be a bit smaller of an increase once we aren't copying redundant SVG files.

Reply:
Will set it to 655KB